src/mbgl/layout/symbol_layout.cpp

```diff
@@ -298,8 +297,7 @@ void SymbolLayout::addFeature(const GeometryCollection &lines,
             // be drawn across tile boundaries. Instead they need to be included in
             // the buffers for both tiles and clipped to tile boundaries at draw time.
             //
-            // TODO remove the `&& false` when is #1673 implemented
-            const bool addToBuffers = (mode == MapMode::Still) || inside || (mayOverlap && false);
+            const bool addToBuffers = inside || mayOverlap;
```

Comment:
I don't see how removing `mode == MapMode::Still` here is safe. In still mode, rendering a single tile, we _cannot_ throw away features in the tile buffer under any circumstances, for the reasons detailed in https://github.com/mapbox/mapbox-gl-native/issues/6670#issuecomment-254331003.